README.md

```diff
@@ -231,15 +231,19 @@ and for Ivy:
 <dependency org="io.reactivex.rxjava2" name="rxjava" rev="x.y.z" />
 ```
 
-Snapshots are available via [JFrog](https://oss.jfrog.org/webapp/#/home):
+Snapshots are available via [JFrog](https://oss.jfrog.org/webapp/#/home) in [libs-snapshot/io/reactivex/rxjava2/rxjava/](https://oss.jfrog.org/libs-snapshot/io/reactivex/rxjava2/rxjava/)
 
 ```groovy
 repositories {
     maven { url 'https://oss.jfrog.org/libs-snapshot' }
 }
 
 dependencies {
+    // for versions 2.0.0 - 2.1.4
```

Comment:
I guess having 2 snapshots in this case kinda defeats purpose of snapshots, 

Snapshot just always point to latest build, so I'd just update `2.0.0-DP0-SNAPSHOT` to `2.2.0-SNAPSHOT` (and maybe change snapshot artifact name to `2.x-SNAPSHOT` in a separate PR